pkgs/development/ocaml-modules/ppx_import/default.nix

```diff
@@ -21,9 +20,6 @@ buildDunePackage rec {
     ppx_tools_versioned ocaml-migrate-parsetree
   ];
 
-  doCheck = true;
```

Comment:
Didn't `ppxlib` 0.22 make `ppx_import`'s tests again? Or am I remembering wrong.

Reply:
Fixed the tests for all versions by overriding `ppx_deriving` (as for `elpi`).